src/CurlHandler.php

```diff
@@ -11,6 +11,29 @@
  * A handler that uses cURL to send the request.
  */
 class CurlHandler implements HttpHandlerInterface {
+    /**
+     * @var curl handle to be (re)used
```

Comment:
Note the doc block bug as found [here](https://scrutinizer-ci.com/g/vanilla/garden-http/code-structure/feature/add-keep-alive/class/Garden%5CHttp%5CCurlHandler).